src/System.Runtime/tests/Performance/Perf.Double.cs

```diff
@@ -63,41 +88,59 @@ public static IEnumerable<object[]> ToStringWithFormat_TestData()
         {
             string[] formats =
             {
-                "R",
-                "G",
-                "G17",
-                "E",
-                "F50"
+                "R",    // Roundtrip
```

Comment:
Do we still want to put some precision numbers on these?

Reply:
I think it is a good scenario to cover, but we can't realistically do it with benchmark.net adding 10s per "scenario".

Reply:
As is, however. We are basically covering 6, 9, 15, and 17 digits with the current scenarios (between both the Single and Double perf tests).